plugins/archlinux/README.md

```diff
@@ -52,6 +54,8 @@ upgrades were available. Use `pacman -Que` instead.
 
 | Alias   | Command                                         | Description                                                             |
 |---------|-------------------------------------------------|-------------------------------------------------------------------------|
+| auclean | `sudo aura -Sc`                                 | Clean out old and unused caches and packages                            |
+| auclr   | `sudo aura -Scc`                                | Remove all files from the cache                                         |
```

Comment:
[`aura` is fully compatible with `pacman`, they mean the same as the `pacman` aliases](https://fosskers.github.io/aura/pacman.html#clearing-your-package-cache).